src/runtime/mod.rs

```diff
@@ -29,20 +29,35 @@ pub(crate) enum AsyncRuntime {
 
 impl AsyncRuntime {
     /// Spawn a task in the background to run a future.
```

Comment:
We should update this docstring to reflect why it's returning an optional / how it's different from `execute`.

Reply:
Updated